### ISSUE TITLE
Mojave OS Git reinstall

### ISSUE THREAD

**Issue by dirediredocks** [STAT545-UBC/Discussion#523]
Hi, updated to the new Mojave OS for Mac and Git didn't work, so I followed all the steps and reinstalled from scratch. Got everything working, but my directory under the 'Git' tab on Rstudio only shows the .rproj file, not the pulled files from GitHub (not even the README.md). The files, including the README.md, are physically present in the working directory specified working directory though. In other words, I can pull but not push. Thanks!

**Comment by ericjsanders** [comment]
Silly question perhaps, but did you double check that you had the right project open? You would do this by closing your project and opening the exact .Rproj file in the proper working directory.

Sometimes if I use "open recent project", I'll accidentally click on the wrong one, and not realize that I'm working in a different directory. This is especially possible if my project names are similar or I've misnamed one accidentally when I made it.

If this simple solution doesn't help, I'm not sure what technical problem might be happening =(

**Comment by dirediredocks** [comment]
Yeah, restarted R and new session. Deleted previous project folders, made a new one from scratch, opened the .Rproj in the new folder specifically, and still the issue persists. Maybe the 'Git' tab is looking at some other folder? I can still pull edits from GitHub and they land on the new folder, but the 'Git' Rstudio tab is not reading them.

**Comment by aleurcelay** [comment]
Hi @dirediredocks. This happened to me days ago and answered here: https://github.com/STAT545-UBC/Discussion/issues/518
Hope it works for you as well :)
